packages/core/src/components/axes/toolbar.ts

```diff
@@ -180,9 +208,11 @@ export class Toolbar extends Component {
 		enteringOverflowMenuControls
 			.merge(overflowMenuControls)
 			.classed('bx--overflow-menu-options__option--disabled', (d) =>
-				d.shouldBeDisabled()
+				d.shouldBeDisabled ? d.shouldBeDisabled() : false
+			)
+			.attr('aria-disabled', (d) =>
+				d.shouldBeDisabled ? d.shouldBeDisabled() : false
```

Comment:
Looks like lower in the code we're setting a default for `d.shouldBeDisabled`, right? Then do we need the check?